config/config.go

```diff
@@ -269,7 +270,7 @@ type Local struct {
 	EnableProfiler bool `version[0]:"false"`
 
 	// TelemetryToLog records messages to node.log that are normally sent to remote event monitoring
-	TelemetryToLog bool `version[5]:"true"`
```

Comment:
we should not modify versions... they denote config version the param first appeared and its default value

Reply:
reverted.